### PR TITLE
prep for next version by changing previous version

### PR DESCRIPTION
**Pull request by chrisahl**
Signed-off-by: Chris Ahl <cahl@redhat.com>

To get ready for the next release we need to change previous version number to 0.3.3

### CODE REVIEW

**Comment by chrisahl** [comment]
/hold 

Need to answer a few questions on precommit before we merge

**Review by robinbobbitt — APPROVED** [review]

**Pull request merged**
